ironfish/src/evm/decodeOpcodes.ts

```diff
@@ -0,0 +1,56 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+
```

Comment:
What is this file for? Doesn't look like it is doing anything relative to the custom opcodes. I also don't see where it is being used, maybe I am missing something though.